apps/app/src/views/routes/Prescription.tsx

```diff
@@ -495,6 +501,67 @@ export const Prescription = () => {
                 </Text>
               )}
             </HStack>
+
+            {prescription?.fills && prescription?.fills?.length === 0 ? null : (
+              <>
+                <Divider />
+
+                <Text color="gray.500" fontWeight="medium" fontSize="sm">
+                  Orders
+                </Text>
+                <VStack spacing={4} w="full">
+                  {prescription?.fills?.map((fill: Maybe<Fill>) => {
+                    if (!fill) return null;
+                    const address = fill?.order?.pharmacy?.address;
+                    const addressString = address
+                      ? `${address?.street1}, ${address?.city}, ${address?.state} ${address?.postalCode}`
+                      : '';
+                    return (
+                      <LinkBox key={fill.id} w="full" style={{ textDecoration: 'none' }}>
+                        <Card
+                          variant="outline"
+                          p={[2, 3]}
+                          w="full"
+                          shadow="none"
+                          _hover={{
+                            backgroundColor: 'gray.50'
+                          }}
+                        >
+                          <HStack justifyContent="space-between">
+                            <VStack alignItems="start">
+                              <HStack>
+                                <LinkOverlay href={`/orders/${fill?.order?.id}`}>
+                                  <HStack spacing={2}>
+                                    <Text fontSize="md">
+                                      {fill?.order?.pharmacy?.name || 'Pharmacy not set'}
```

Comment:
I think we can make this a little clearer that a pharmacy ahs not been selected
<img width="1248" alt="Screen Shot 2023-07-18 at 10 54 03 AM" src="https://github.com/Photon-Health/client/assets/8848487/07d8e1fc-b890-4def-a236-2cab028d00c4">
since moped often loses track and is looking to select orders for the patient (to be added).